viadot/task_utils.py

```diff
@@ -462,6 +462,22 @@ def df_clean_column(
     return df
 
 
+@task
+def concat_dfs(dfs: List[pd.DataFrame]):
+    """
+    Task to combine list of data frames into one
+
+    Args:
+        dfs (List[pd.DataFrame]): List of dataframes to concat.
+    Returns:
+        full_df (pd.DataFrame()): Pandas dataframe containing all columns from dataframes from list.
+    """
+    full_df = pd.DataFrame()
+    for df in dfs:
+        full_df = pd.concat([full_df, df], axis=1)
+    return full_df
```

Comment:
You forgot to implement my comment, the standard way to do it is to do `pd.contact[dfs]`, or is there any specific reason to do it in a loop?

Reply:
It's already changed, you can see it in the "Files changed". I don't know why here is an old version.